gui/system/hook.py

```diff
@@ -112,11 +112,18 @@ def hook_app_tabs_system(self, request):
         license = get_license()[0]
         if license is not None and not notifier().is_freenas():
             support = models.Support.objects.order_by('-id')[0]
-            tabs.insert(12, {
+            tabs.insert(13, {
```

Comment:
I've made this change because in tabs it's `| Proactive Support | Support |` while in menu it's `Support; Proactive Support`, I suppose someone forgot to change this when he was adding a new tab